modules/nw-pod-network-connectivity-check-object.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+//
+
+[id="nw-pod-network-connectivity-check-object_{context}"]
+= PodNetworkConnectivityCheck object fields
+
+The `PodNetworkConnectivityCheck` object fields are described in the following tables.
+
+.PodNetworkConnectivityCheck object fields
+[cols="2,1,3a",options="header"]
+|===
+
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the object in the following format: `<source>-to-<target>`. The destination described by `<target>` includes one of following strings:
+
+* `load-balancer-api-external`
+* `load-balancer-api-internal`
+* `kubernetes-apiserver-endpoint`
+* `kubernetes-apiserver-service-cluster`
+* `network-check-target`
+* `openshift-apiserver-endpoint`
+* `openshift-apiserver-service-cluster`
+
+|`metadata.namespace`
+|`string`
+|The namespace that the object is associated with. This value is always `openshift-network-diagnostics`.
+
+|`spec.sourcePod`
+|`string`
+|The name of the pod where the connection check originates, such as `network-check-source-596b4c6566-rgh92`.
+
+|`spec.targetEndpoint`
+|`string`
+|The target of the connection check, such as `api.devcluster.example.com:6443`.
```

Comment:
Is this always a server? If so, should we mention that?

Reply:
@mburke5678, it's either an API server, LB, or pod managed by a daemon set on a node. So not exclusively a server.